scripts/CreateMirrorHierarchy.pl

```diff
@@ -156,15 +161,26 @@ sub CanextendCheck {
 # Make sure the file system driver module is loaded on the template
 @output = `modprobe $fsType >/dev/null 2>&1`;
 
-# Create the mirror resource on the template system (the source)
-system "$LKDRBIN/create -t $LKDRTAG -s $SWITCHBACK -p ${opt_l}1 -h \"$NETRAIDTYPE\" -n $mountPoint -x \"\" -e \"\" -f $mountPoint -y $fsType -a $syncType -b $BITMAP -z \"no\" -w \"\"";
 
-$ret = $? >> 8;
+my $localPartition = "${opt_l}1";
+
+#if we have resource instances for the mirror, skip trying to create the mirror
+my $inslist_ret = system("$LKBIN/ins_list -R $mountPoint > /dev/null 2>&1");
+if($inslist_ret){
+	print "Resource instance for filesystem resource with tag \"$mountPoint\" does not exist, creating resource\n";
```

Comment:
Should this print still be here? Same goes for other statements. Is this how we get them into the cloud-init log (or wherever this is tracked)? If so it's fine. Just making sure this isn't debug

Reply:
Yea the output is just redirected to /var/log/cloud-init.log so this is how we get it in there. This isnt debug